adyen.go

```diff
@@ -97,13 +103,13 @@ func (a *Adyen) SetCurrency(currency string) {
 }
 
 // execute request on Adyen side, transforms "requestEntity" into JSON representation
-func (a *Adyen) execute(method string, requestEntity interface{}) (*Response, error) {
+func (a *Adyen) execute(method string, service string, requestEntity interface{}) (*Response, error) {
 	body, err := json.Marshal(requestEntity)
 	if err != nil {
 		return nil, err
 	}
 
-	url := a.adyenURL(method)
+	url := a.adyenURL(service, method) + "/" + service + "/"
```

Comment:
Not sure if `service` should be added here. You already passing `service` to `adyenURL`  and it should return a URL like this `https://pal-test.adyen.com/pal/servlet/Recurring/v25/listRecurringDetails` or `https://pal-test.adyen.com/pal/servlet/Payment/v25/Capture`

Reply:
Good catch I should have review a bit more the code ;-)